OutOfSchool/OutOfSchool.EmailSender/ServiceProviderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OutOfSchool.EmailSender
+{
+    public static class ServiceProviderExtensions
+    {
+        public static IServiceCollection AddEmailSender(
+            this IServiceCollection services,
+            Action<SmtpOptions> options)
+        {
+            services.AddSingleton<IEmailSender, EmailSender>();
+            if (options == null)
```

Comment:
remember we were talking about an ability to disable e-mail config for local testing? Maybe add an implementation to the sender service method `Task SendAsync(Message message)` that if the config is missing or some option is specified to disable sending the message, then it just prints it to `log.Debug` so we can copy the link or verify email content locally.